includes/GbWeiss.php

```diff
@@ -80,6 +76,11 @@ public function initOptionPage(): void
             ->addOption(new Option('Client Secret', 'client_secret', __('Client Secret', self::$languageDomain), 'account', 'string'));
 
         $optionsPage->addTab($accountTab);
+        $orderStatuses = $this->orderStateRepository->getAllOrderStates();
+
+        $fulfilmentTab = new FulfilmentOptionsTab($orderStatuses);
```

Comment:
fulfillment :)

Reply:
find and replace it, since the typie is also in the name "validateFulfilmentSetting"